ansible/copy_files_from_ctrl_node.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+scp ex4:/home/ec2-user/guillem/* .
+git add .
+git commit -m "$1"
```

Comment:
I'm not sure why we've got a git step here - do we actually want to commit changes? Why?

If I were to guess, this is because git is throwing errors when you try to override some changes - if that's the case then using the `force` parameter on the git checkout step feels more specific to what we're actually trying to achieve than this!

Reply:
I created this script to copy the files from the remote node and commit them into git... ./copy_files_from_ctrl_node.sh "commit message blah blah blah"